src/WidenImage.css

```diff
@@ -0,0 +1,61 @@
+
+.wideImageOverride {
```

Comment:
Ohh thanks for the BEM link... I'll try to read up on that tomorrow.

As far as naming conventions go I'm pretty open to suggestions. Good chat topic for tomorrow.